se.data/langs/es_la.json

```
@@ -0,0 +1,171 @@
+{
+  "help_pagination_prefix": "c --------------- ",
+  "help_sel_wand_tooltip": "g Usar el comando wand para cambiarlo",
+  "help_header_title": "d Ayuda de Scarpet-Edit",
+  "help_cmd_brush_paste": "l Registar una brocha para pegar el portapapeles con origen en el bloque seleccionado",
+  "help_cmd_flood_tooltip": "g usar [axis] Para realizar un llenado plano",
+  "no_redo": "r Sin acciones que rehacer para el jugador %s",
+  "help_cmd_expand": "l Expande selección [magn] respecto a [pos]",
+  "brush_info_no_flags": "w sin banderas",
+  "help_make_selection": "c Usar la barita para seleccionar posiciones inicial y final",
+  "brush_empty_list": "gi Sin brochas registradas hasta ahora",
+  "action_cone": "cono",
+  "action_cuboid": "cuboide",
+  "help_cmd_brush_list": "l Lista todas las prochas registradas y sus acciones",
+  "action_cylinder": "cilindro",
+  "angel_block_clear": "w Desregistra el bloque ángel",
+  "help_pagination_prev": "g Página anterior (%d)",
+  "help_cmd_copy": "l Copiar selección actual al portapapeles",
+  "help_cmd_brush_flood": "l Registra brocha para realizar un llenado de inundación con [block] empezando en el bloque seleccionado",
+  "more_moves_redo": "w Valor demasiado grande, rehaciando todas las acciones de %s",
+  "no_undo_history": "w Sin historia de acciones para mostrar de %s",
+  "action_outline": "contorno",
+  "help_cmd_brush_feature": "l Registra brocha para colocar decoraciones",
+  "action_stack": "apilado",
+  "help_cmd_brush_info": "l Da información detallada de la brocha en mano",
+  "help_header_prefix": "c ----------------- [ ",
+  "help_selected_wand": "c Varita seleccionada",
+  "action_prism_polygon": "polígono_prisma",
+  "action_line": "línea",
+  "action_structure_paste": "pegado_estructura",
+  "brush_list_header": "bc === Las brochas actuales son ===",
+  "help_cmd_brush_cylinder": "l Registra brocha para crar un cilindro con [radius] y [height] a lo largo de [axis] de [block]",
+  "help_cmd_brush_spray": "l Registra brocha para pintura en aerosol con el radio y la cantidad de puntos dados",
+  "more_moves_undo": "w Valor demasiado grande, deshaciando todas las acciones de %s",
+  "help_welcome_tooltip": "y Síii!",
+  "action_item_angel": "item de bloque ángel",
+  "saved_structure": "w Estructura guardada como %s.nbt",
+  "help_cmd_rotate": "l Rota [deg] en torno a [pos]",
+  "brush_info_action": "b  Acción: ",
+  "help_cmd_brush_pyramid": "l Registra brocha para crear pirámide con [radius] y [height] sobre eje [axis] en la dirección del signo",
+  "langs_completeness": "gi Nota: %s la traducción está completa en un %s%%: %s frases faltantes",
```

```suggestion
  "langs_completeness": "gi Nota: La traducción al %s solo está completa en un %s%%: %s frases faltantes",
```
First %s refers to the language name, so `La traducción al Español solo está completa...`

---

woops, I really missed this on XD